patterns/minimising-data-size-transmitted.md

```diff
@@ -0,0 +1,38 @@
+# Minimising the size of data transmitted
+
+## Version
+1.0
+
+## Submitted By
+Sarah Hsu
+
+## Published Date
+TBD
+
+## Intent
+Minimising the size of data transmitted
+
+## Tags
+Cloud, Networking, Cloud Engineer, Small
+
+## Problem
+From an energy-efficiency perspective, it's better to minimise the size of the data transmitted so that less energy is required because the network traffic is reduced. 
+
+## Solution
+Minimise the size of data transmitted by compressing files or payloads.
```

Comment:
Is this pattern specifically on compression or is this meant to handle minimizing the dataset itself (i.e. not sending properties or values that are not needed or can be correlated later)?

Reply:
It's a bit of both. I have split them into two patterns.